ibm/service/power/data_source_ibm_pi_volume.go

```diff
@@ -125,32 +128,31 @@ func dataSourceIBMPIVolumeRead(ctx context.Context, d *schema.ResourceData, meta
 		return diag.FromErr(err)
 	}
 
-	cloudInstanceID := d.Get(helpers.PICloudInstanceId).(string)
+	cloudInstanceID := d.Get(Arg_CloudInstanceID).(string)
 	volumeC := instance.NewIBMPIVolumeClient(ctx, sess, cloudInstanceID)
 	volumedata, err := volumeC.Get(d.Get(helpers.PIVolumeName).(string))
 	if err != nil {
 		return diag.FromErr(err)
 	}
 
 	d.SetId(*volumedata.VolumeID)
-	d.Set("size", volumedata.Size)
-	d.Set("state", volumedata.State)
-	d.Set("shareable", volumedata.Shareable)
-	d.Set("bootable", volumedata.Bootable)
-	d.Set("disk_type", volumedata.DiskType)
-	d.Set("volume_pool", volumedata.VolumePool)
-	d.Set("wwn", volumedata.Wwn)
-	d.Set("replication_enabled", volumedata.ReplicationEnabled)
-	d.Set("replication_type", volumedata.ReplicationType)
-	d.Set("replication_status", volumedata.ReplicationStatus)
-	d.Set("auxiliary", volumedata.Auxiliary)
-	d.Set("consistency_group_name", volumedata.ConsistencyGroupName)
-	d.Set("group_id", volumedata.GroupID)
-	d.Set("mirroring_state", volumedata.MirroringState)
-	d.Set("primary_role", volumedata.PrimaryRole)
-	d.Set("auxiliary_volume_name", volumedata.AuxVolumeName)
-	d.Set("master_volume_name", volumedata.MasterVolumeName)
-	d.Set("io_throttle_rate", volumedata.IoThrottleRate)
```

Comment:
Why `io_throttle_rate` is removed?

Reply:
This was a merge error. I'll fix it also.